src/com/hms_networks/americas/sc/taginfo/TagInfoManager.java

```diff
@@ -546,7 +549,8 @@ private static boolean convertStrToBool(String stringBool) {
    * @param tagId the Ewon tag ID
    * @return {@link TagInfo} object associated with the given tag ID
    */
-  public static synchronized TagInfo getTagInfoFromTagId(int tagId) throws IllegalStateException, IndexOutOfBoundsException{
+  public static synchronized TagInfo getTagInfoFromTagId(int tagId)
```

Comment:
formatting change in a commit about javadocs. This change was probably intended to go/should go in the previous commit.

Reply:
:/ you are right

Reply:
fixed